.github/workflows/ci.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
           mkdir -p /tmp/myof/success
           mkdir -p /tmp/myof/error
           mkdir -p /tmp/myof/attachment
-          ../dist/myof-client_linux_amd64_v1/myof-client --config $CONFIG_DIR/config.properties &
+          ../dist/edi-client_linux_amd64_v1/edi-client --config $CONFIG_DIR/config.properties &
```

Comment:
Sicher das das hier der richtige Name ist? Im installer selber wird er noch mit myof-client.exe referenziert?

Reply:
No this is the followup, needs to be changed back. Done